src/debugger/nodeDebugWrapper.ts

```diff
@@ -43,6 +43,7 @@ interface ILaunchArgs {
     platform: string;
     target?: string;
     internalDebuggerPort?: any;

```

Comment:
If VS Code doesn't guarantee any kind of type validation, then we should mark all the parameters as any, and make sure that we handle them if we are not what we expect them to be.